dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldFilterService.java

```diff
@@ -65,6 +70,22 @@
     @Qualifier( "jsonMapper" )
     private final ObjectMapper jsonMapper;
 
+    private final SchemaService schemaService;
+
+    private final AclService aclService;
+
+    private final CurrentUserService currentUserService;
+
+    public FieldFilterService( FieldPathHelper fieldPathHelper, ObjectMapper jsonMapper, SchemaService schemaService,
```

Comment:
could this use `@AllArgsConstructor` when you annotate the `@Qualifier` on the field?

Reply:
But still need to configure the jsonMapper (it does local customization on it)